### DISCUSSION TITLE
Is SwiftGen dead?

### DISCUSSION THREAD

**Discussion by Arideno** [SwiftGen/SwiftGen#1142] [General]
Hello,

The last update was over six months ago, and new pull requests are not reviewed. I assume the project has no maintainers. Does anyone have any information on this?

**Comment by dioKaratzas** [comment]
@AliSoftware Thanks for all your work on SwiftGen! Could you share if there are any plans for future updates or if the project is looking for maintainers?

**Comment by AliSoftware** [comment]
Hi there,

Sorry for the long silence.
I confirm that I don't have the bandwidth and free time to take care and maintain SwiftGen anymore those days.

If there's any volunteer interested to step up as maintainer, I am thus interested.
@djbe has also helped a bit as a co-maintainer in the past, in case he can help and/or have maintainer suggestions.

**Reply by dioKaratzas** [reply]
Hi @AliSoftware,
I’d be happy to volunteer as a maintainer and help keep SwiftGen updated. Could you share any next steps or requirements for stepping into the maintainer role?
